cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,94 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
+import { settings } from 'cluster';
+import faker from 'faker';
+import SettingsPageObgect from '../support/pages/settings.pageObject';
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import { fa } from 'faker/lib/locales';
+import HomePageObject from '../support/pages/home.pageObject';
+import ProfilePageObject from '../support/pages/profile.pageObject';
+
+const testData = {
+  email: faker.internet.email().toLowerCase(),
+  username: faker.name.firstName().toLowerCase(),
+  password: faker.internet.password(),
+  bio: faker.lorem.words()
+};
+
+const settingsPageObgect = new SettingsPageObgect();
+const signInPageObject = new SignInPageObject();
```

Comment:
```suggestion
const signInPage = new SignInPageObject();
```
rename your constants like I did it